benchmark/single-source/FloatingPointConversion.swift

```diff
@@ -61,7 +61,13 @@ extension MockBinaryFloatingPoint {
   init(_ value: Int) { self.init(_Value(value)) }
   init(_ value: Float) { self.init(_Value(value)) }
   init(_ value: Double) { self.init(_Value(value)) }
-  init(_ value: Float80) { self.init(_Value(value)) }
+  #if canImport(Darwin) || os(Linux)
```

Comment:
I believe the compilation condition we use elsewhere is:

```
#if !(os(Windows) || os(Android)) && (arch(i386) || arch(x86_64))
```

Reply:
Ah, I was looking at FloatingPointParsing and etc.